### PR TITLE
P0 features

### PR DESCRIPTION
**Pull request by git-0r**
## Features: 
- Focus of day
- weather
- quote on every refresh

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+function setInStorage(obj) {
+  chrome.storage.local.set(obj);
+}
+
+function setWallpaper(key) {
+  chrome.storage.local.get(key, async (result) => {
+    const cache = result[key];
+    if (cache) {
+      const randomNum = Math.ceil(Math.random() * (79 - 0) + 0);
```
Comment (irajbhinde) [inline index.js:11]: Couldn't understand why you're using (79-0) + 0, can you please explain?

File: index.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        chrome.storage.local.set({ _xUsername: username });
+        nameForm.classList.remove("set_visible_flex");
+        onboard();
+      });
+    }
+  });
+}
+
+function focus() {
+  const d = new Date();
```
Comment (irajbhinde) [inline index.js:131]: You can use a better naming here

**Review by ark2002 — APPROVED** [review]
looks great good to go

**Review by irajbhinde — COMMENTED** [review]
The code looks real clean, kudos! 

Some UX-related suggestions, you can implement if it suits you, as per the screenshots shared, it felt like the weather & time containers are too big, some spacing would look good. Also, welcome message on center would look better

File: index.html [hunk 3]
```diff
@@ -5,11 +5,55 @@
...
+        <p class="name_content"></p>
+        <div class="weather_wrapper">
+            <div class="container">
+                <p class="desc"></p>
+                <img src="" class="weather_image" alt="weather image" />
+            </div>
+            <div class="container">
+                <div>
+                    <p class="temp"></p>
+                    <span class="degree_symbol">&#176;</span>
```
Comment (dev-ashishsethi) [inline index.html:23]: You must have a fallback when using unicode because in some browser it is not recognized

**Review by dev-ashishsethi — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (git-0r) [reply index.js:11]: api gives 80 wallpapers in every request, so saving them in localstorage then using any one random on refresh

File: index.html (hunk 3 above)
Comment (git-0r) [reply index.html:23]: thanks, i'll keep this in mind and update accordingly.

**Pull request merged**